jiant/preprocess.py

```diff
@@ -305,8 +310,10 @@ def build_tasks(args):
 
     # 3) build / load word vectors
     word_embs = None
-    if args.input_module not in ["elmo", "gpt", "scratch"] and not args.input_module.startswith(
-        "bert"
+    if (
```

Comment:
similar to above: `args.input_module in ['glove', 'fasttext']`. This list will probably grow a lot slower than the contextual word embedding models.

Reply:
On it.